README.md

```diff
@@ -10,59 +10,93 @@ dotnet add package Jusibe
 
 ## Usage
 
-A Client class provides three public methods for accessing the API. Instantianting the class is as given below:
+First, in your `appsettings.json` file, create a new section for Jusibe as follows:
```

Comment:
I hope there's still a way to use this in a non asp.net core environment.

Can the README reflect both ways?